genai/client_test.go

```diff
@@ -49,7 +49,12 @@ func TestLive(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer client.Close()
+	defer func(client *Client) {
```

Comment:
We don't usually check the result of client.Close; it isn't really actionable. An error here doesn't affect anything, since the service is stateless.